a4-speakup/assets/scss/_variables.scss

```diff
@@ -826,9 +826,9 @@ $progress-bar-transition:           width .6s ease !default;
 // List group
 
 $list-group-bg:                     $white !default;
-$list-group-border-color:           rgba($black, .125) !default;
+$list-group-border-color:           none !default;
 $list-group-border-width:           $border-width !default;
-$list-group-border-radius:          $border-radius !default;
+$list-group-border-radius:          0 !default;
```

Comment:
Same here: only used here? Then it's fine!
What do general buttons have? Is that in the border-radius variable then?